src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,10 +1,17 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
 
 public class MovieSessionRequestDto {
+    @NotNull
+    @Min(value = 1)
     private Long movieId;
```

Comment:
[The primitive type boolean can not hold a null value. If you don't assign any value, it will default to 0 which maps to false.](https://stackoverflow.com/questions/54386835/how-to-make-not-null-primitive-fields-in-spring-jpa#:~:text=The%20primitive%20type%20boolean%20can%20not%20hold%20a%20null%20value.%20If%20you%20don%27t%20assign%20any%20value%2C%20it%20will%20default%20to%200%20which%20maps%20to%20false.)